Vindicta.Altis/Templates/Factions/AAF.sqf

```diff
@@ -127,6 +127,6 @@ _array set [T_DRONE, _drone];
 _array set [T_CARGO, _cargo];
 _array set [T_GROUP, _group];
 _array set [T_NAME, "tAAF"];
-
+_array set [T_FACTION, T_FACTION_Military];
```

Comment:
Nice, I actually had it in mind to add faction entries to automate listing of factions in the UI.

Reply:
Yeah we can replace the other "faction" strings etc with this for some consistency as well. e.g. Garrison faction type is a string at the moment.